Loop/Managers/LoopDataManager.swift

```diff
@@ -357,6 +447,9 @@ class LoopDataManager {
         } else {
             recommendedTempBasal = nil
         }
+
+        recommendedBolus = nil
+        recommendedBolus = try self.recommendBolus()
```

Comment:
I see WatchDataManager is also asking for current recommended bolus; that wouldn't make sense as a part of LoopStatus, would it?